### ISSUE TITLE
Action items & notes from BICCN phone call 11/22/2019

### ISSUE THREAD

**Issue by hertzron** [nemoarchive/analytics#79]
Overall very positive feedback for the utility of NeMO Analytics. 
Requests to prioritize:
- changing coloring of expression to monchromatic on the static tSNE as the middle range (white) is confusing --> Joshua
- Match the cluster coloring to the BICCN established colors for different levels of clusters (3 leves of clustering, and have all three levels available per dataset) --> Brian
- Make gene-specific link-outs to Eran's epigenetic tool
- Import epigenetic data to Epiviz --> Brian/Seth/Jayaram

**Comment by jkanche** [comment]
I think the last time Alex helped me on this, we generated bigwigs for 3 clusters of 10 samples each from rna and atac. I was under the impression that we will have a Trackhub setup for datasets going forward and Epiviz reads files from this hub ?

**Comment by mukamel-lab** [comment]
Thanks for presenting today, Ronna - Indeed we are very excited about these tools.  

Adding to your list: For the color scheme matching a good first step would be to ensure that when you display multiple datasets showing the same types of cells (e.g. all the BICCN transcriptome datasets), you should use the same colors for clusters/cell types with the same name. Currently it looks like this is not the case. Eventually we can provide the color scheme that we'll use for the paper, but internal consistency is the most important thing.

**Comment by hertzron** [comment]
Agreed. This may require building 'color palattes' by organ/system e.g.,
'BICCN', 'EAR' etc or 'custom' which may work so long the convention to
name a cell type is consistent.

**Comment by mukamel-lab** [comment]
Maybe there could be some kind of (optional) unique ID for cell types supplied by the uploader. When the unique ID is present in multiple datasets, it coordinates the display so they have matching colors. This might be better than relying on the name, which could "accidentally" match across unrelated datasets?

**Comment by jorvis** [comment]
If you have any existing color palette conventions you could send along
(and, ideally, a dataset to go with it) I'd like to see them in order to
better plan an implementation here.

Joshua

**Comment by mukamel-lab** [comment]
Hi Joshua -

See the attached draft figure that has a color scheme for each cluster.  There is also this:

https://www.dropbox.com/home/BICCN%20MiniBrain%20JointDataAnalysis/Mini-Atlas%20Publication%202019/Transcriptome_jointAnalysis?preview=subclass_colors.csv
[cid:275039C2-D9A8-4A4B-9D47-B217A1BE0DE4@ucsd.edu][cid:F595577F-1FA6-4B3C-AAED-FE87F4328E3B@ucsd.edu]
Eran A. Mukamel
Assistant Professor of Cognitive Science
University of California, San Diego
emukamel@ucsd.edu<mailto:emukamel@ucsd.edu>
http://brainome.ucsd.edu



On Nov 25, 2019, at 8:18 AM, Joshua Orvis <notifications@github.com<mailto:notifications@github.com>> wrote:

If you have any existing color palette conventions you could send along
(and, ideally, a dataset to go with it) I'd like to see them in order to
better plan an implementation here.

Joshua

**Comment by jorvis** [comment]
Hi, I recently completed work adding support for custom cluster coloring, but when I followed your link to grab the csv file it says it is no longer present.  Could you possibly attach it here instead?

**Comment by mukamel-lab** [comment]
Is this what you mean?


#00FF66 CR
#DA808C Lamp5
#D633FF Sncg
#B864CC Vip
#FF9900 Sst
#D93137 Pvalb
#C4EC04

L2/3 IT
#50B2AD L5 IT
#A19922 L6 IT
#0D5B78 L5 ET
#5100FF L6 IT Car3
#3E9E64 L5/6 NP
#2D8CB8 L6 CT
#53377D L6b
#665C47 Astro
#2E3E39 Oligo
#8D6C62 Endo
#807059 SMC
#665547 Peri
#697255 VLMC
#94AF97 Macrophage

Eran A. Mukamel
Assistant Professor of Cognitive Science
University of California, San Diego
emukamel@ucsd.edu<mailto:emukamel@ucsd.edu>
http://brainome.ucsd.edu



On Jan 27, 2020, at 11:27 AM, Joshua Orvis <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi, I recently completed work adding support for custom cluster coloring, but when I followed your link to grab the csv file it says it is no longer present. Could you possibly attach it here instead?

**Comment by jorvis** [comment]
Yes that should work nicely.

**Comment by mukamel-lab** [comment]
Hi Joshua - I see that your name is on the author list for the BICCN paper but there’s no email or affiliation listed. Could you please enter these?

https://docs.google.com/spreadsheets/d/1WPV9xqI516REptn9Z6zUrCrxqW5Qyu9oPJMU2l-ZJ5U/edit?usp=sharing

Thanks,
Eran

Eran A. Mukamel
Assistant Professor of Cognitive Science
University of California, San Diego
emukamel@ucsd.edu<mailto:emukamel@ucsd.edu>
http://brainome.ucsd.edu

On Jan 27, 2020, at 11:27 AM, Joshua Orvis <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi, I recently completed work adding support for custom cluster coloring, but when I followed your link to grab the csv file it says it is no longer present. Could you possibly attach it here instead?
